### ISSUE TITLE
gdal broken with openjpeg update

### ISSUE THREAD

**Issue by tonytheodore** [mxe/mxe#1917]
gdal configure only looks for openjpeg in 2.0 and 2.1 subdirectories - the [recent update to 2.2](https://github.com/mxe/mxe/commit/acb344645b47dd1e9fb71c5b8bfa4d88d19b4f72) causes it to fail.

There's a fix in #1889 that will hopefully be merged soon - this is just a placeholder in case anyone runs into the issue.

**Comment by lawipac** [comment]
any way to reverse back to 2.1 for a smooth build?

**Comment by lawipac** [comment]
at the moment I have no way to have a workable mxe build.

**Comment by tonytheodore** [comment]
From #1889, the latest `gdal` now supports later versions of `openjpeg` (the other patches are for new/updated pkgs). You can do:
```
make update-package-gdal
rm src/gdal-1-fixes.patch
make gdal
```
in the interim to get a working build.

**Comment by tonytheodore** [comment]
Fixed in #1926.
